nautilus_trader/adapters/interactive_brokers/historical/client.py

```diff
@@ -60,7 +62,8 @@ def __init__(
         client_id: int = 1,
         market_data_type: MarketDataTypeEnum = MarketDataTypeEnum.REALTIME,
         log_level: str = "INFO",
-        cache: Cache | None = None,
+        cache_database: CacheDatabaseFacade | None = None,
```

Comment:
We could also set this up in a [similar way to the kernel](https://github.com/nautechsystems/nautilus_trader/blob/develop/nautilus_trader/system/kernel.py#L298) by using the `database` attribute which is already on `CacheConfig`?